backend/migrations/20201107133524_add_fullname_to_students.js

```diff
@@ -0,0 +1,15 @@
+const { tableStudents } = require('../src/models/student')
+
+exports.up = function(knex, Promise) {
+    return knex.schema.table(tableStudents, (table) => {
+        table.text('FullName').notNullable().defaultTo('fullname');
```

Comment:
Hi Laura, yes, we have changed the defaultTo to an empty string instead. We kept the defaultTo to prevent the migration rollback errors if we were to change the table schema. Hope that works for you :)

Reply:
@xuanjane makes sense. @lauratsui you can update your approval status to Approved.